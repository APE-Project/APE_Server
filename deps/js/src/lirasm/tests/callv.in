; test call to void function

forty_two = immi 42
callv printi cdecl forty_two
reti forty_two
