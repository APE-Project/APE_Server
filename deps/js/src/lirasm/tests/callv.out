42
Output is: 42
